gulpfile.js

```diff
@@ -334,39 +334,39 @@ function dist_rollup() {
 
 // Create runable app directories in ./apps
 function apps(done) {
-    var platforms = getPlatforms();
+    const platforms = getPlatforms();
     removeItem(platforms, 'android');
 
     buildNWAppsWrapper(platforms, 'normal', APPS_DIR, done);
 }
 
 function listPostBuildTasks(folder, done) {
 
-    var platforms = getPlatforms();
+    const platforms = getPlatforms();
 
-    var postBuildTasks = [];
+    const postBuildTasks = [];
 
-    if (platforms.indexOf('linux32') != -1) {
-        postBuildTasks.push(function post_build_linux32(done) {
+    if (platforms.indexOf('linux32') !== -1) {
+        postBuildTasks.push(function post_build_linux32() {
             return post_build('linux32', folder, done);
         });
     }
 
-    if (platforms.indexOf('linux64') != -1) {
-        postBuildTasks.push(function post_build_linux64(done) {
+    if (platforms.indexOf('linux64') !== -1) {
+        postBuildTasks.push(function post_build_linux64() {
```

Comment:
This seems to be wrong - we are actually referencing the `done` parameter in the innermost scope - in the line just below it. It probably just needs a different name to distinguish it from the outer scope.

Reply:
Good catch, I ended simply removing the outer scope parameter. It was not used at all.